src/test/java/org/opensearch/knn/index/LuceneEngineIT.java

```diff
@@ -449,4 +516,37 @@ private void validateQueryResultsWithFilters(
                 .containsAll(expectedDocIdsKLimitsFilterResult)
         );
     }
+
+    private void validateRadiusSearchResults(
+        final float[] searchVector,
+        final float radius,
+        final SpaceType spaceType,
+        final int expectedResults,
+        @Nullable final String filterField,
+        @Nullable final String filterValue
+    ) throws Exception {
+        KNNQueryBuilder queryBuilder;
```

Comment:
I think for integ tests, its more readable to just construct the request as JSON as opposed to using `KNNQueryBuilder` which is implemented in the server

Reply:
Looks like existing IT tests are mainly using searchKNNIndex which need require in `KNNQueryBuilder`